lib/features/authentication/presentation/widgets/bottom_nav_widget.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:resto_admin/core/themes/app_theme.dart';
+import 'package:resto_admin/core/widget/under_construction_widget.dart';
+import 'package:resto_admin/features/authentication/presentation/providers/bottom_provider.dart';
+
+class BottomNaviWidget extends ConsumerWidget {
+  BottomNaviWidget({super.key});
+
+  final pageController = PageController();
```

Comment:
use the hook for page controller, or use stateful widget and dispose the controller